### PR TITLE
Add applications proof of serving notice template link and updated proposal map label/sublabel (ALCS-1336)

### PR DESCRIPTION
**Pull request by trslater**
- Add link to proof of serving notice template
- Change/add label/sublabel for proposal map field
  - Added to TUR proposal, review, view, ALCS, and public

### CODE REVIEW

File: portal-frontend/src/app/features/applications/edit-submission/proposal/tur-proposal/tur-proposal.component.html
Comment (urmikgov) [inline portal-frontend/src/app/features/applications/edit-submission/proposal/tur-proposal/tur-proposal.component.html:1]: is this the text to be replaced?

Comment (trslater) [reply portal-frontend/src/app/features/applications/edit-submission/proposal/tur-proposal/tur-proposal.component.html:1]: Are you referring to "FIXME:ALC website"?

Comment (urmikgov) [reply portal-frontend/src/app/features/applications/edit-submission/proposal/tur-proposal/tur-proposal.component.html:1]: yes

Comment (trslater) [reply portal-frontend/src/app/features/applications/edit-submission/proposal/tur-proposal/tur-proposal.component.html:1]: I had to double-check this as well, but it is intended. Apparently, it to be replaced later.

**Review by urmikgov — APPROVED** [review]

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**